### PR TITLE
Rodolpho's Card

### PR DESCRIPTION
**Pull request by rodolphonapoleon**
Added my card information to the index.html file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,46 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:118]: Please change this comment to 
`<!-- ________ Your Name Contributor card START ________ -->`

File: index.html [hunk 2]
```diff
@@ -115,6 +115,46 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="A platform to learn">
+                  W3Schools
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:156]: Please change this comment to 
`<!-- ________ Your Name Contributor card END________ -->`

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @rodolphonapoleon, thank you for contributing to this project.
You need to make a couple of simple changes to your code before we go ahead and merge your pull request.
Please change these two comments, they help everyone to identify the cards of each contributor. 
Once you made those changes, get back to us. We will be able to review and merge your pull request in a flash.

**Comment by Preet-Sojitra** [comment]
@rodolphonapoleon Your current pull request is causing around 40 merge conflicts. So you are requested to open a new pull request, once you open new pull request, we will merge it.
Thank you :)

**Comment by Preet-Sojitra** [comment]
> I fixed my issue

Merge conflicts aren't solve yet. Please open new pull request. In order to open new pull request, you need to make new branch, and then commit your changes, because one branch can open only one pull request.

**Comment by win21H2** [comment]
Hello @rodolphonapoleon, I will be closing this issue since it has 100+ conflicts although I will manually add it to the website :)

**Pull request closed without merging**